src/components/organisms/RegisterForm/RegisterForm.md

```diff
@@ -4,10 +4,21 @@ import { ImageModal, ThemeTester } from "../../../styleguide";
 import { EscolaLMSContextProvider } from "@escolalms/sdk/lib/react/context";
 import { Text } from "../../../";
 
+const fieldLabels = {
+  email: <strong>Overwrite email label</strong>,
+  "AdditionalFields.Privacy Policy": (
+    <p>
```

Comment:
@qunabu I would use the component <Text> or something, to make sure there would be no font problem

Reply:
good point